src/glass/src/app/shared/services/local-inventory.service.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Project Aquarium's frontend (glass)
+ * Copyright (C) 2021 SUSE, LLC.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import {
+  Inventory,
+  LocalNodeService,
+  Nic,
+  NodeStatus,
+  Volume
+} from './api/local.service';
+import { PollService } from './poll.service';
+
+export type NICEntry = {
+  name: string;
+  info: Nic;
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalInventoryService {
+
+  private subject: BehaviorSubject<Inventory|undefined> =
```

Comment:
More see in the `getInventory` method comment.
```suggestion
  private inventorySource: BehaviorSubject<Inventory|undefined> =
```